superchain/Dockerfile

```diff
@@ -1,29 +1,42 @@
 FROM amazonlinux:2
 
+# Install deltarpm as it can speed up the upgrade processes
+RUN yum -y install deltarpm
+
 # Install .NET Core, mono & PowerShell
 ENV DOTNET_CLI_TELEMETRY_OPTOUT=true                                                                                    \
-    DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true
```

Comment:
Why are we removing this? Is this because people are doing repeated builds on the same container, so the long-term optimization cost is worth it?

Reply:
I have actually looked at how Microsoft built *their* `dotnet` images, and the way they go past the "fist time experience" is by invoking `dotnet help` (which I have added).

Reply:
Yeah, I noticed where you probably got these from, and checked out the dockerfile. The difference is that that image is designed for development as well, why this may just be used for automation.

The performance impact is probably negligible - but just wanted to give some weight to what these options did, rather than simply adding/removing them.